app/views/reports/itemized_distributions/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= render(
+  "shared/filtered_card",
+  id: "purchases",
+  gradient: "secondary",
+  title: "Itemized Distributions",
+  subtitle: @selected_date_range,
+  type: :table,
+  filter_url: reports_itemized_distributions_path
+) do %>
+
+  <% if @itemized_distribution_data.empty? %>
+    <div class="alert alert-warning" role="alert">
+      No itemized distributions found for the selected date range.
+    </div>
+  <% else %>
+    <table class="table table-hover striped text-left">
+      <thead>
+        <tr>
+          <th>&nbsp;&nbsp;&nbsp;Item</th>
```

Comment:
Heh these &nbsp;'s remind me of the early ought's. Can you use a CSS style instead to indent?